### PR TITLE
Wizard recipe: IOAPI-v3.2.0

### PR DESCRIPTION
**Pull request by ctessum**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: IOAPI
* Version: v3.2.0

@staticfloat please review and merge.

### CODE REVIEW

File: I/IOAPI/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+cp Makefile.nocpl Makefile
+cp Makeinclude.Linux2_x86_64gfort Makeinclude.$BIN
+
+make fixed_src
+make # Parallel make (-j > 1) does not work
+
+cp fixed_src/* ${includedir} # FORTRAN .EXT (include) files
+
+gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive) -lnetcdf -Wl,$(flagon --no-whole-archive) -lnetcdff
```
Comment (giordano) [inline I/IOAPI/build_tarballs.jl:40]: Try this (the `flagon` script has a bug on macOS at the moment :persevere:)
```suggestion
gfortran -shared -fPIC -fopenmp -o ${libdir}/libioapi.${dlext} -Wl,$(flagon --whole-archive) ${BINDIR}/libioapi.a -Wl,$(flagon --no-whole-archive | cut -d' ' -f1) -lnetcdf -lnetcdff
```

File: I/IOAPI/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+sources = [
+    GitSource("https://github.com/cjcoats/ioapi-3.2.git", "ef5d5f4e112c249b593b19426421f25d79ae094b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add tcsh # build script is in csh
+
+cd $WORKSPACE/srcdir/ioapi-3.2/ioapi/
+export HOME=${WORKSPACE}/srcdir
```
Comment (giordano) [inline I/IOAPI/build_tarballs.jl:18]: Need to install the license manually:
```suggestion
install_license LICENSE
export HOME=${WORKSPACE}/srcdir
```

File: I/IOAPI/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    ExecutableProduct("m3interp", :m3interp),
+    ExecutableProduct("m3mask", :m3mask),
+    ExecutableProduct("mtxcalc", :mtxcalc)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="NetCDF_jll", uuid="7243133f-43d8-5620-bbf4-c2c921802cf3"))
+    Dependency(PackageSpec(name="NetCDFF_jll", uuid="78e728a9-57fe-5d11-897c-5014b89e5f84"))
+]
```
Comment (giordano) [inline I/IOAPI/build_tarballs.jl:128]:
```suggestion
    Dependency(PackageSpec(name="CompilerSupportLibraries_jll", uuid="e66e0078-7015-5450-92f7-15fbd957f2ae"))

]
```

File: I/IOAPI/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+sources = [
+    GitSource("https://github.com/cjcoats/ioapi-3.2.git", "ef5d5f4e112c249b593b19426421f25d79ae094b")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add tcsh # build script is in csh
+
+cd $WORKSPACE/srcdir/ioapi-3.2/ioapi/
+install_license LICENSE
```
Comment (giordano) [inline I/IOAPI/build_tarballs.jl:18]: Meh, wrong directory:
```suggestion
install_license ../LICENSE
```

**Comment by ctessum** [comment]
Thanks! I have another question about this, however. This library comes with [fortran include files with the .EXT extension](https://cjcoats.github.io/ioapi/INCLUDE.html) such as [this one](https://github.com/cjcoats/ioapi-3.2/blob/master/ioapi/fixed_src/PARMS3.EXT), that downstream programs need to compile against. In the build script, I [copied](https://github.com/JuliaPackaging/Yggdrasil/pull/2814/files#diff-eac4872648a3dc162d1c83826e3f51a4e9cc5608a520827f61ca409ffbd0a085R39) the .EXT files into ${includedir}, but they don't show up in the tarball. Is there a way to include these files?

Thanks for your help!

**Comment by giordano** [comment]
> but they don't show up in the tarball.

Uh?

```console
% wget -qO - 'https://github.com/JuliaBinaryWrappers/IOAPI_jll.jl/releases/download/IOAPI-v3.2.0%2B0/IOAPI.v3.2.0.x86_64-linux-gnu-libgfortran5.tar.gz' | tar tzvf - | grep "include/"
-rw-r--r-- 0/0            4153 1970-01-01 01:00 include/ATDSC3.EXT
-rw-r--r-- 0/0            6952 1970-01-01 01:00 include/CONST3.EXT
-rw-r--r-- 0/0           12031 1970-01-01 01:00 include/FDESC3.EXT
-rw-r--r-- 0/0           17905 1970-01-01 01:00 include/IODECL3.EXT
-rw-r--r-- 0/0           50804 1970-01-01 01:00 include/NETCDF.EXT
-rw-r--r-- 0/0            1396 1970-01-01 01:00 include/NOTICE.EXT
-rw-r--r-- 0/0            9246 1970-01-01 01:00 include/PARMS3.EXT
-rw-r--r-- 0/0            9629 1970-01-01 01:00 include/STATE3.EXT
```

**Comment by ctessum** [comment]
Ah, I see, got it! My fault for not actually checking the tarball :)

**Pull request merged**